pages/community/overview.mdx

```diff
@@ -14,19 +14,11 @@ canonical: "/community/overview"
 - [Forum](https://forum.rescript-lang.org)
 - [ReScript GitHub Org](https://github.com/rescript-lang/)
 - [ReScript YouTube Channel](https://www.youtube.com/@rescriptlang)
-- [ReScript Online Meetup on Guild](https://guild.host/rescript-online-meetup)
```

Comment:
There hasn't been an event posted here since end of 2023, so I think we should hide it unless we have it active.